.github/workflows/release.yml

```diff
@@ -8,15 +8,15 @@ on:
 jobs:
   release:
     name: Release
-    runs-on: macOS-latest
+    runs-on: macOS-14
     steps:
       - name: Check out repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
-      - name: Use Xcode 14.1
-        run: sudo xcode-select -switch /Applications/Xcode_14.1.app
+      - name: Use Xcode 15.1
+        run: sudo xcode-select -switch /Applications/Xcode_15.1.app
```

Comment:
We will want to use 15.0.1 to support Carthage merchants using Xcode 15.0 ([slack convo here](https://paypal.slack.com/archives/C03U9SEQYHK/p1711464753326179?thread_ts=1711387809.750689&cid=C03U9SEQYHK)). Up to you on if we want to update everything to 15.0.1 or just this portion. It looks like the macOS-14 runner should support this: https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md#xcode

Reply:
thanks for letting me know @jaxdesmarais! updated [here](b27be70ff88fe05890851e2aeb4e242bc052eddf)